### DISCUSSION TITLE
Δήλωση πρόθεσης προαιρετικής παρουσίασης ΟΣΣ 3

### DISCUSSION THREAD

**Discussion by std159312** [sdy60/syllabus#123] [Q&A]
Καλημέρα σας, εκ μέρους της ομάδας https://github.com/orgs/SDY60-2023/teams/e-netteam δηλώνω την πρόθεσή μας να παρουσιάσουμε το μέρος Δ αύριο 21/1

[3η ΓΕ παρουσίαση(e-netTeam).pdf](https://github.com/sdy60/syllabus/files/13998421/3.e-netTeam.pdf)

**Comment by Maurphal** [comment]
Δηλωνω και τη δικιά μας προθεση . ευχαριστώ!
[Παρουσίαση ΕΡΓΑΣΙΑΣ 3.pdf](https://github.com/sdy60/syllabus/files/14001577/3.pdf)
